docs/tutorial.md

```diff
@@ -394,7 +394,107 @@ What's going on here? Let's step through the code.
 
 (sec_extracting_individuals)=
 
-## Extracting particular SLiM individuals
+## Extracting SLiM individuals
+
+### Extracting a sample of individuals
+
+Often we would like to sample a number of individuals from a population.
+This section provides an example workflow and draws
+attention to some easy-to-miss details for this approach.
+
+Let's start with a nonWF simulation similar to what we've seen previously.
+
+```{literalinclude} sample.slim
+```
+
+```{code-cell}
+%%bash
+slim -s 1 sample.slim
+```
+
+The first, most common method to extract individuals is simply to get all
+those that were alive at a particular time,
+using {meth}`SlimTreeSequence.individuals_alive_at`. For instance, to get
+the list of individual IDs of all those alive at the end of the
+simulation (i.e., zero time units ago), we could do:
+
+```{code-cell}
+orig_ts = pyslim.load("sample.trees")
+alive = orig_ts.individuals_alive_at(0)
+
+print(f"There are {len(alive)} individuals alive in the final generation.")
+```
+
+These are individual IDs. One way to take a sample of living individuals
+and write their SNPs to a VCF is:
+
+```{code-cell}
+np.random.seed(1)
+keep_indivs = np.random.choice(alive, 100, replace=False)
+ts = pyslim.SlimTreeSequence(msprime.mutate(orig_ts, rate=1e-8, random_seed=1))
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=keep_indivs)
+```
+
+
+### Extracting individuals after simplification
+
+In cases where we want to obtain genotypes *after* simplification, we
+need to pay attention to details concerning individual nodes. As
```

Comment:
I think we should avoid saying "individual nodes", since it confuses "individuals" and "nodes", which are different. What did you mean here?

Reply:
good point. I meant specific nodes